src/db/tables/subscribers.ts

```diff
@@ -455,18 +484,18 @@ async function getSubscribersWaitingForMonthlyEmail (options = {}) {
  */
 // Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
 /* c8 ignore start */
-async function updateMonthlyEmailTimestamp (email) {
-  const res = await knex('subscribers')
+async function updateMonthlyEmailTimestamp(email: string) {
```

Comment:
Nit, but might as well:

```suggestion
async function updateMonthlyEmailTimestamp(email: SubscriberRow["primary_email"]) {
```